docs/source/configure.rst

```diff
@@ -17,10 +17,10 @@ When you run the command to start the Quipucords server, you supply values for s
 
 The following steps guide you through those choices.
 
-1. Accept or change the default exposed server port to use for HTTPS communication. By default, the server exposes port 443, which is the standard HTTPS port. You can choose to use that port or remap the port to be used on your server.
+1. Accept or change the default exposed server port to use for HTTPS communication. By default, the server exposes port 9443, which is the standard HTTPS port. You can choose to use that port or remap the port to be used on your server.
 
-   - If you select to expose port 443, you would use the following option when you run the Docker command to start the server: ``-p 443:443``.
-   - If you want to remap the port on your system, you would supply a new value for the port when you run the Docker command to start the server. The syntax of this option is  ``-p <host_port>:<container_port>``. For example, to remap the port to ``8443``, you would enter the following option in the command: ``-p 8443:443``. Additionally, Docker supplies an option to select a free port for all exposed ports by using the ``-P`` option; the port mapping is then available from the ``sudo docker ps`` command.
+   - If you select to expose port 9443, you would use the following option when you run the Docker command to start the server: ``-p 9443:443``.
+   - If you want to remap the port on your system, you would supply a new value for the port when you run the Docker command to start the server. The syntax of this option is  ``-p <host_port>:<container_port>``. For example, to remap the port to ``9443``, you would enter the following option in the command: ``-p 9443:443``. Additionally, Docker supplies an option to select a free port for all exposed ports by using the ``-P`` option; the port mapping is then available from the ``sudo docker ps`` command.
```

Comment:
The second part of this is talking about remapping the port (not using the default) so we should use a value that is not the default, you could leave it at 8443.